### DISCUSSION TITLE
NgRx Store in browser memory

### DISCUSSION THREAD

**Discussion by EzioArqam** [ngrx/platform#3309] [Q&A] [answered]
We are working on a project where we would like to open multiple angular apps from a central .NET application, 
we have some concerns regarding the performance of the store of these multiple apps if they will each have their own independent store, if so, where exactly is the store saved, in session storage or in local storage? 
and how can we view the store other than the Redux dev tools so as to know how much memory the store is consuming?

**Comment by timdeschryver** [comment, marked as answer]
The store doesn't local/session storage, but all of the entities are stored in memory.
I suppose you can calculate it by using the memory profiler or by subscribing to the store to select the whole state tree.
That being said, I would be amazed if this causes to be a problem.

**Reply by EzioArqam** [reply]
Thanks for making this clear to me :)
